examples/image_classifier/requirements.txt

```diff
@@ -0,0 +1,2 @@
+torchvision
```

Comment:
Oh I see, sorry that you had to go through this. To be honest, it would be great to have a single pip install command to install everything in one go. Sadly, pip has its limitation and so we're forced to install dependencies in other means. In our case that is `python ./ts_scripts/install_dependencies.py`. This happens in the first step of the getting started guide:
![image](https://github.com/user-attachments/assets/decb668c-1f45-47c6-95e8-0a6c419a65bf)
It installs some non Python dependencies and selects the correct requirements.txt. Did the command return any error for you?